index.js

```diff
@@ -24,6 +24,7 @@
  */
 
 (function (root, factory) {
+    'use strict';
```

Comment:
Wouldn't you get the same effect by putting this at the very top of the file?

Reply:
that's the way I had it initially, but after webpacking it gets stuck at the top of the generated file, which in theory would apply strict mode to all of our webpacked dependencies. 'use strict' is [function scoped](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) so doing it this way applies the rule to semotus only.

Sifting through our webpacked files it looks like there are a few third party libraries that also follow this pattern.